File: server/validations/Email.py
from validate_email import validate_email


class Email:
    @staticmethod
    def validate(email: str) -> bool:
        is_valid = validate_email(email, verify=True)
        return is_valid

    @staticmethod
    def validate_without_verify(email: str) -> bool:
        is_valid = validate_email(email)
        return is_valid

File: server-js/validations/Email.js
const { verifyEmail } = require("@devmehq/email-validator-js");
var verifier = require("@digitalroute/email-verify");

class Email {
  static async validate(email) {
    return await verifyEmail({
      emailAddress: email,
      verifyMx: true,
      timeout: 5000,
    }).then((result) => result.validFormat);
  }

  static async validateWithoutVerify(email) {
    return await verifyEmail({
      emailAddress: email,
      verifyMx: false,
      verifySmtp: false,
      timeout: 5000,
    }).then((result) => result.validFormat);
  }
}

module.exports = Email;
